runtime/libcn/src/utils.c

```diff
@@ -96,10 +96,30 @@ void ghost_stack_depth_incr(void) {
     print_error_msg_info();
 }
 
+
+#define FMT_PTR "\x1b[33m%#lx\x1b[0m"
+// #define KMAG  "\x1B[35m"
+#define FMT_PTR_2 "\x1B[35m%#lx\x1B[0m"
+
 void ghost_stack_depth_decr(void) {
     cn_stack_depth--;
     // update_error_message_info(0);
     print_error_msg_info();
+    // TODO - revisit this Dhruv over-confident change
+    hash_table_iterator it = ht_iterator(cn_ownership_global_ghost_state);
+    printf("CN pointers leaked at (%ld) stack-depth: ", cn_stack_depth);
+    _Bool leaked = false;
+    while (ht_next(&it)) {
+        uintptr_t *key = it.key;
```

Comment:
Can you change this to an `intptr_t`? The type of `it.key` is `signed long` and it's throwing a warning when building CN